src/stylesheets/utility/_tag-list.scss

```diff
@@ -43,6 +45,41 @@
   }
 }
 
+.tag-list__btn {
+  pointer-events: none;
```

Comment:
This threw me for a loop initially because it should totally disable any JS events you have added, but I see below you are `unset`ting them for desktop below. Maybe add a comment here that explains that the buttons are visual only on mobile.

Reply:
Maybe instead of setting and resetting this property, just assign it within a "mobile only" breakpoint using max-width, then you won't have to unset it below.